extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/filters/accesslog/DefaultAccessLogReceiver.java

```diff
@@ -70,60 +74,34 @@ public class DefaultAccessLogReceiver implements AccessLogReceiver, Runnable, Cl
 
     private long changeOverPoint;
     private String currentDateString;
-    private boolean forceLogRotation;
 
     private final Path outputDirectory;
     private final Path defaultLogFile;
 
     private final String logBaseName;
     private final String logNameSuffix; // always starts with a '.' character
 
-    private Writer writer = null;
```

Comment:
reset seems valid due to rotation needs to write on different files.

Reply:
its writing into the same file just making copy of this as rotation. Its writing all the time into one and the same file.